things/api.py

```diff
@@ -112,7 +112,7 @@ def tasks(uuid=None, include_items=False, **kwargs):  # noqa: C901
         Only output length of result. This is done by a SQL COUNT query.
 
     last : str, default None, optional
-        Limit returned tasks to tasks created in the last X days or weeks. Takes as input string of format 'X[d/w]' where X is a non-negative integer and 'd' stands for days and 'w' for weeks.
+        Limit returned tasks to tasks created in the last X days or weeks.
```

Comment:
My intended suggestion was to wrap into multiple lines of at most 80-88 characters each. Your content was great! :)